src/moped/src/main/java/core/behaviour_states/states/AdaptiveCruiseControl.java

```diff
@@ -1,4 +1,24 @@
 package core.behaviour_states.states;
 
+import core.action_strategies.ActionStrategy;
+
 public class AdaptiveCruiseControl implements BehaviourState {
+
+    private ActionStrategy currentStrategy;
+    // Remove?

```

Comment:
Too early to determine if we really need the field, but I vote for leaving it there.